resources/src/main.rs

```diff
@@ -95,8 +102,11 @@ async fn intercept_command(body: &Value) {
         "ping" => commands::ping::run(channel).await,
         _ => log::info!("Invalid command: {:?}", ..),
     }
+    // if text.contains("buns") {
+    //     commands::buns::run(channel, enterprise_user_id).await;
+    // }
     if text.contains("buns") {
-        commands::buns::run(channel, enterprise_user_id).await;
```

Comment:
Is this going to be added back once `buns.rs` is changed to an emoji reaction? I gotta build our buns leaderboard next.

Reply:
Yes, I was using this to test manually instead of waiting for people to join the workspace